.gitignore

```diff
@@ -84,6 +84,8 @@ ios/NativeSigner/Database/Database/*
 # rust
 rust/target
 rust/*/target
+rust/Cargo.lock
```

Comment:
Doesn't look like we should add Cargo.lock to .gitignore ([ref](https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html))

Reply:
We should, see here

https://github.com/paritytech/parity-signer/blob/2fcbe3c23d2d217a11e334d55e11a14605b1a258/scripts/build.sh#L32

https://github.com/paritytech/parity-signer/blob/2fcbe3c23d2d217a11e334d55e11a14605b1a258/scripts/build.sh#L79

This is a hack, for Cargo does not support this target-specific building, and we have different Cargo.toml in final build for different archs, because android requires dynamic JNI interface and ios requires static C lib.

Reply:
Although I'm planning to change this scripting system further soon, to separate build stages and introduce convenient `just` file to manage all this.